JumpTheQueue/index.asciidoc

```diff
@@ -0,0 +1,41 @@
+= Tutorial for the Jump The Queue Application
+====
+Jump The Queue is a tutorial application
+
+More information about JumpTheQueue on https://github.com/devonfw/jump-the-queue
+====
+
+Install Devonfw IDE
+[step]
+--
+installDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+Clone the Jump-The-Queue Repository.
+[step]
+--
+cloneRepository("", "https://github.com/devonfw/jump-the-queue.git")
+--
+
+====
+Build and Run the Java Server
```

Comment:
You can remove the text before here, because you set 'Build and Run the Java Server' as the title. So it would be display twice among each other

Reply:
The title is only for representation in the index.json and not displayed at the katacoda tutorial

Reply:
The title specified in the index.json is displayed at the top of a step. When you open the tutorial on https://katacoda.com/devonfw-dev/scenarios/59_jump-the-queue_jumpthequeue and open step 3 you see what I mean.

Reply:
ah i see